src/client/kmd.ts

```diff
@@ -89,7 +117,14 @@ export class KmdClient extends ServiceClient {
     const req = {
       wallet_handle_token: walletHandle,
     };
-    const res = await this.c.post('/v1/wallet/release', req);
+    const res = await this.c.post({
+      relativePath: '/v1/wallet/release',
+      data: req,
+      parseBody: true,
+      jsonOptions: {
+        intDecoding: IntDecoding.SAFE,
+      },
+    });
```

Comment:
Perhaps `KmdClient` should have a `post` method with `parseBody: true` and the `jsonOptions` set?  Maybe even just take two arguments, `relativePath` and `data`, so all of these stay as trivial as they were before by calling `this.post` instead of `this.c.post`?
I may delete this if/when I notice there are exceptions.

Reply:
Not a bad idea, all of the POST requests here are very similar

Reply:
Added in 80a0c35